app/javascript/dashboard/routes/dashboard/settings/integrations/Webhooks/WebhookForm.vue

```diff
@@ -70,10 +69,67 @@ export default {
         subscriptions: this.subscriptions,
       });
     },
+    getEventNamei18n,
   },
 };
 </script>
 
+<template>
```

Comment:
It seems like there are duplicate `templates`

Reply:
Bad merge, didn't notice it.